### PR TITLE
fix: base price

### PR DESCRIPTION
**Pull request by rbedeoan-plenty**
feature migration from 
https://github.com/plentymarkets/plentyshop-pwa/pull/238/files#diff-b66ab6328920287731b2bbd93090614658297002460a0cd827fa299fa00de92d

had to refactor props to get strings not the actual product

task
https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/71550/
lowest price not necesary

### CODE REVIEW

File: apps/web/components/BasePrice/types.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export type BasePriceProps = {
+  unitContent?: number;
+  unitName?: string;
+  basePrice?: string;
+  oneline: boolean;
+  contentLineFirst: boolean;
```
Comment (ksted) [inline apps/web/components/BasePrice/types.ts:6]: **question**: Do we actually need `contentLineFirst`? In all use cases so far, it gets passed as `true`.

File: apps/web/components/BasePrice/types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export type BasePriceProps = {
+  unitContent?: number;
+  unitName?: string;
+  basePrice?: string;
+  oneline: boolean;
```
Comment (ksted) [inline apps/web/components/BasePrice/types.ts:5]: **thought**: I know that the PBI says the component should be reusable, but looking at the props, I'm not sure that's really the way to go. The component is small (and will get even smaller without the conditions), so we could also have a `BasePrice` and a `BasePriceInline`.

My gut feeling is that with props, we should distinguish between props that supply data and props that do nothing but create conditions inside the template.

Would welcome further discussion here @plentymarkets/shop-front-end.
(Obviously this is not blocking in any way, we can refactor it later on.)

File: apps/web/components/BasePrice/BasePrice.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+.twolineItems #basePrice {
+  width: 100%;
+}
+
+#lineSeparator {
+  margin-right: 2px;
+  margin-left: 2px;
+}
+</style>
```
[the remark this answers is not in the document]
Comment (rbedeoan-plenty) [reply apps/web/components/BasePrice/BasePrice.vue:64]: css cases are very custom and using tailwind here is not that easy...

File: apps/web/components/BasePrice/types.ts (hunk 2 above)
Comment (rbedeoan-plenty) [reply apps/web/components/BasePrice/types.ts:5]: i also thought this was a good idea but timp said he wanted 1 component with divs switching from css

File: apps/web/components/BasePrice/types.ts (hunk 1 above)
Comment (rbedeoan-plenty) [reply apps/web/components/BasePrice/types.ts:6]: Screenshot 2023-08-23 at 11 13 12

Comment (rbedeoan-plenty) [reply apps/web/components/BasePrice/types.ts:6]: if you will have cart sidebar and want to respect initial design you just switch it to false
https://github.com/plentymarkets/plentyshop-pwa/pull/238/files#diff-5d1c1af86406377af1bd6219832c64ac513e996076afd808bce19a4867bc0983

Comment (rbedeoan-plenty) [reply apps/web/components/BasePrice/types.ts:6]: Tim said i should have functionality exactly as in pwa

File: apps/web/lang/en.json [hunk 4]
```diff
@@ -49,6 +49,8 @@
   "closeListSettings": "Close list settings",
   "clearFilters": "Clear all filters",
   "showProducts": "Show products",
+  "Content": "Content",
+  "Baseprice": "Baseprice",
```
Comment (ksted) [inline apps/web/lang/en.json:53]:
```suggestion
  "content": "Content",
  "basePrice": "Base price",
```

Comment (ksted) [reply apps/web/lang/en.json:53]: **todo**: Please also add the German translations in `de.json`.
```
"content": "Inhalt",
"basePrice": "Grundpreis"
```

File: apps/web/components/BasePrice/BasePrice.vue [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="text-sm" v-if="basePrice">
+    <div>{{ $t('Content') }}: {{ unitContent }} {{ unitName }}</div>
+    <div>{{ $t('Baseprice') }}: {{ basePrice }}</div>
```
Comment (ksted) [inline apps/web/components/BasePrice/BasePrice.vue:4]:
```suggestion
    <div>{{ $t('content') }}: {{ unitContent }} {{ unitName }}</div>
    <div>{{ $t('basePrice') }}: {{ basePrice }}</div>
```

**Review by ksted — CHANGES_REQUESTED** [review]

**Review by ksted — APPROVED** [review]

**Pull request merged**